File: app/src/main/java/com/firstapp/robinpc/tongue_twisters_deluxe/view/AboutActivity.java
package com.firstapp.robinpc.tongue_twisters_deluxe.view;

import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.bumptech.glide.load.engine.DiskCacheStrategy;
import com.firstapp.robinpc.tongue_twisters_deluxe.R;

public class AboutActivity extends AppCompatActivity {

    ImageView imageView1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        setupActionBar();

        imageView1 = (ImageView) findViewById(R.id.imageView1);
        Glide.with(this)
                .load("https://2.bp.blogspot.com/-uj-cptq9ELE/WITQa8VA1xI/AAAAAAAAADg/In0g-bhXoUscy1xR8PTjdueG8PjOTgi3gCLcB/s1600/IMG-20161006-WA0009.jpg")
                .diskCacheStrategy(DiskCacheStrategy.ALL)
                .crossFade()
                .fitCenter()
                .into(imageView1);
    }

    private void setupActionBar() {
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            // Show the Up button in the action bar.
            actionBar.setDisplayHomeAsUpEnabled(true);
        }
    }

}

File: app/src/main/java/com/firstapp/robinpc/tongue_twisters_deluxe/about_screen/AboutActivity.kt
package com.firstapp.robinpc.tongue_twisters_deluxe.about_screen

import android.annotation.SuppressLint
import android.content.Context
import android.os.Build
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v4.content.ContextCompat
import com.bumptech.glide.Glide
import com.bumptech.glide.request.RequestOptions
import com.firstapp.robinpc.tongue_twisters_deluxe.R
import com.firstapp.robinpc.tongue_twisters_deluxe.utils.AppConstants
import io.github.inflationx.viewpump.ViewPumpContextWrapper
import kotlinx.android.synthetic.main.activity_about.*

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        setup()
    }

    private fun setup() {
        setStatusBarColor()
        loadProfilePictures()
        setClickListeners()
    }

    private fun setClickListeners() {
        go_back.setOnClickListener {
            onBackPressed()
        }
    }

    private fun setStatusBarColor() {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
            window.statusBarColor = ContextCompat.getColor(this, R.color.colorPrimaryIntensity3)
    }

    @SuppressLint("CheckResult")
    private fun loadProfilePictures() {
        val options = RequestOptions()
        options.centerCrop()
        Glide.with(this).load(AppConstants.URL_AMIT_CONTENT_CREATOR_PROFILE)
                .apply(options).into(amit_image)
        Glide.with(this).load(AppConstants.URL_ANUBHAV_CONTENT_CREATOR_PROFILE)
                .apply(options).into(anubhav_image)
        Glide.with(this).load(AppConstants.URL_ROBIN_DEVELOPER_PROFILE)
                .apply(options).into(robin_image)
    }

    override fun onBackPressed() {
        super.onBackPressed()
        overridePendingTransition(R.anim.slide_in_left_activity, R.anim.slide_out_right_activity)
    }

    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(ViewPumpContextWrapper.wrap(newBase))
    }
}
